### PR TITLE
improves deployment docs and scripts

### PR DESCRIPTION
**Pull request by andkononykhin**

### CODE REVIEW

File: docs/running-node.md [hunk 1]
```diff
@@ -150,40 +167,53 @@ Run
...
+*   the script assumes that:
+    *   current user is going to be used for `dcld` service to run as
+    *   current user is in sudoers list
+*   if it's not acceptable for your case please consult a less automated guide [Running Genesis Node](running-genesis-node.md)
+
 This command:
 
 *   generates a new key entry for a node admin account
 *   generates `genesis.json` file with the following entries:
     *   a genesis account for the key entry above with `Trustee` and `NodeAdmin` roles
```
Comment (ashcherbakov) [inline docs/running-node.md:181]: We have two separate accounts now

File: docs/running-node.md [hunk 2]
```diff
@@ -150,40 +167,53 @@ Run
...
+*Notes*
+
+*   the script assumes that:
+    *   current user is going to be used for `dcld` service to run as
+    *   current user is in sudoers list
+*   if it's not acceptable for your case please consult a less automated guide [Running Validator Node](running-validator-node.md)
+
 This command:
 
 *   generates a new key entry for a node admin account
```
Comment (ashcherbakov) [inline docs/running-node.md:219]: It doesn't generate any keys anymore, does it?

File: docs/running-node.md [hunk 3]
```diff
@@ -218,9 +241,9 @@ $ dclcli tx validator add-node \
 
 If the transaction has been successfully written you would find `"success": true` in the output JSON.
 
-#### Notify other validator administrators
+#### Notify other validator admins
 
-Provide the node's `id`, `ip` and peer port (by default `26656`) to other validator administrators
+Provide the node's `id`, `ip` and peer port (by default `26656`) to other validator admins.
```
Comment (ashcherbakov) [inline docs/running-node.md:246]: Makes sense to clarify how to get the node's id.

File: docs/running-node.md (hunk 1 above)
Comment (andkononykhin) [reply docs/running-node.md:181]: yes, thanks

File: docs/running-node.md (hunk 2 above)
Comment (andkononykhin) [reply docs/running-node.md:219]: yes, missed place, thanks

File: docs/running-node.md (hunk 3 above)
Comment (andkononykhin) [reply docs/running-node.md]: updated

**Review by ashcherbakov — APPROVED** [review]

**Pull request merged**